### PR TITLE
add accountable person and annual premium to items

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/models/item.go [hunk 1]
```diff
@@ -403,3 +407,109 @@ func (i *Items) ConvertToAPI(tx *pop.Connection) api.Items {
...
+		if err := policyDependent.FindByID(tx, i.PolicyDependentID.UUID); err != nil {
+			panic("error finding policy dependent " + i.PolicyDependentID.UUID.String())
+		}
+		return policyDependent.Name
+	}
+	panic("no accountable person defined for item " + i.ID.String())
+}
+
+func (i *Item) GetAnnualPremium() int {
+	return (i.CoverageAmount + 25) / 50 // 2% rounded to the nearest cent
```
Comment (Baggerone) [inline application/models/item.go:430]: What is the `+ 25` for?

**Review by Baggerone — APPROVED** [review]

File: application/models/item.go (hunk 1 above)
Comment (Schparky) [reply application/models/item.go:430]: To cause it to round up if the resulting fraction is greater than 0.5. Simply dividing by 50 will cause a truncation of the fractional part.

Comment (Baggerone) [reply application/models/item.go:430]: :+1:

File: application/api/users.go [hunk 2]
```diff
@@ -30,6 +30,9 @@ type User struct {
 	// last name
 	LastName string `json:"last_name"`
 
+	// full name
+	Name string `json_"name"`
```
Comment (fillup) [inline application/api/users.go:34]:
```suggestion
	Name string `json:"name"`
```

**Pull request merged**